app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SubscriptionsController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_after_action :verify_authorized
+
+  def new; end
```

Comment:
Instead of skipping `verify_authorized`, use `authorize Subscription` on `#new` and allow anyone to access it in the policy